### ISSUE TITLE
March update: Several buttons missing labels, accessibility severely broken

### ISSUE THREAD

**Issue by Menelion** [microsoft/vscode#147139]
I use VS Code Insiders. Just got a report from a user that in the latest update buttons like New File and New Folder in the Explorer view are not labeled anymore. Just updated myself and I definitely can confirm that with JAWS 2022.

If you go to the Explorer view (**Ctrl**+**Shift**+**E**) and tab through the buttons, you'll hear just "button button" instead of the correct labels.

That does break accessibility in quite a severe way. CC @isidorn

**Comment by velimudrvelimudr** [comment]
I will add.
1. It's hardly a matter of labels. NVDA buttons reads correctly. Problem when working with JAWS 2021, 2022.
2. Buttons in all panels are not readable. For example, the same is observed with the "Structure" section and in general in all sections that are displayed in my interface.
I apologize for the google translation. I don't write in English :-(

**Comment by Menelion** [comment]
Thanks @velimudrvelimudr! Really had no time for a good thorough testing, I must travel today. Anyway @isidorn it's a VS Code problem as it was totally OK in the previous version.

**Comment by adns44** [comment]
I can confirm this problem with March 2022 update. I use JAWS2022, and usualy I develop in docker. The button labels are missing which makes some development processes less smooth.

I must use workarounds, e.g. navigate to the parent folder in explorer, open the context menu and click to new file instead use new file button that label is invisible with the mentioned screen reader.

**Comment by isidorn** [comment]
Thanks for filling this, but it sounds like a duplicate of https://github.com/microsoft/vscode/issues/146658
So let's close this issue and continue the discussion there
